source/fundamentals/auth.txt

```diff
@@ -229,145 +228,179 @@ connect using ``MONGODB-CR``.
+- As environment variables
+- In a ``MongoClient`` configuration
 
+The next sections demonstrate how to specify your credentials each way
+and use the following placeholders:
 
-The following code snippets show how to specify the authentication mechanism,
-using the following placeholders:
+* ``awsKeyId`` - value of your AWS access key ID
+* ``awsSecretKey`` - value of your AWS secret access key
+* ``atlasUri`` - network address of your MongoDB Atlas deployment
+* ``hostname`` - hostname of your MongoDB Atlas deployment
+* ``port`` - port of your MongoDB Atlas deployment
+* ``awsSessionToken`` - value of your AWS session token *(optional)*
 
-* ``awsKeyId`` - value of your ``AWS_ACCESS_KEY_ID``.
-* ``awsSecretKey`` - value of your ``AWS_SECRET_ACCESS_KEY``.
-* ``atlasUri`` - network address of your MongoDB Atlas instance.
-* ``awsSessionToken`` - value of your ``AWS_SESSION_TOKEN``. *(optional)*
+.. _java-mongodb-aws-env-variables:
 
-.. important:: URL-encode Your Credentials
+Specify Your Credentials in Environment Variables
+++++++++++++++++++++++++++++++++++++++++++++++++++
 
-   Make sure to URL-encode your credentials to prevent backslash or other
-   characters from causing parsing errors. The following code example
-   shows you how to URL-encode a sample string, represented by the placeholder
-   ``fieldValue``:
+You can provide your AWS IAM credentials by instructing the driver to
+use the ``MONGODB-AWS`` authentication mechanism and by setting the
+appropriate environment variables.
 
-   .. literalinclude:: /includes/fundamentals/code-snippets/MongoDbAwsAuth.java
-      :language: java
-      :dedent:
-      :start-after: start urlEncode
-      :end-before: end urlEncode
+.. tip:: Add the AWS SDK for Java
 
-Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
-tab below for instructions and sample code for specifying this authentication
-mechanism:
+   When you use environment variables to specify your AWS IAM credentials, we
+   recommend adding the AWS SDK for Java (v1 or v2) dependency to your
+   project. In addition to authentication, the SDK offers the following
+   features:
 
-.. tabs::
+   - Credential caching which helps your application avoid rate limiting
+   - Credential provider management for use with the `Elastic Kubernetes Service <https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html>`__.
 
-   .. tab::
-      :tabid: Connection String
+   To use the AWS SDK, include the dependencies for it in your project and set
+   the credentials for one of the authentication methods in the provider
+   chain.
 
-      To specify the ``MONGODB-AWS`` authentication mechanism using a
-      connection string, assign the ``authMechanism`` parameter the value
-      ``"MONGODB-AWS"`` in your connection string. Your code to instantiate
-      a ``MongoClient`` should look something like this:
+   If your project includes AWS SDK for Java **v2**, the driver passes the
```

Comment:
> the driver passes the credentials

The driver passes nothing.  The credentials are obtained by the SDK from the environment